docs/UsersGuide/api/cmake/support/Utils.md

```diff
@@ -79,6 +79,6 @@ hashes.txt. This allows for asserts on file ID not string.
 
 Prints the dependency list of the module supplied as well as the include directories.
 
-- **MODULE_NAME**: module name to print dependencies fors
+- **MODULE_NAME**: module name to print dependencies
```

Comment:
I think `for` was intended, but ending this phrase with a preposition is awkward and the word doesn't really add much

Reply:
Agreed.